### ISSUE TITLE
Support TB 72 daily

### ISSUE THREAD

**Issue by jdgeenen** [jdgeenen/gcontactsync#150]
**Is your feature request related to a problem? Please describe.**
Support TB 72 daily.  There are some massive UI issues.

**Comment by stepand76** [comment]
I just upgraded to beta channel (71.0b1, 72.0b1) and realized that gContactSync (3.1.7) stopped working. Any chance that it will be supported?

**Comment by alexarnaud** [comment]
@jdgeenen Any news on this? Do you plan to support TB 78 (the next stable) at the end of this month/in July?

**Comment by SuperSentient** [comment]
I agree. Are there any plans to support Thunderbird 78? Please support TB 78.
